src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -267,7 +267,7 @@ public static Type[] GetTypeArray(object[] args)
             for (int i = 0; i < cls.Length; i++)
             {
                 if (args[i] == null)
-                    throw new ArgumentNullException();
+                    throw new ArgumentException(SR.ArgumentNull_ArrayValue);
```

Comment:
Since this call is not specifying paramName I don't understand why the analyzer isn't firing on it.  Seems like it should be

```suggestion
                    throw new ArgumentException(SR.ArgumentNull_ArrayValue, nameof(args));
```

Reply:
good catch, i might built without turning the analyzer on. For now it is still `(<Rule Id="CA2208" Action="Info" />)`